ansible/library/reduce_and_add_sonic_images.py

```diff
@@ -74,6 +75,12 @@ def install_new_sonic_image(module, new_image_url):
         exec_command(module, cmd="umount /tmp/tmpfs", ignore_error=True)
         exec_command(module, cmd="rm -rf /tmp/tmpfs", ignore_error=True)
 
+    # If sonic device is configured with minigraph, remove config_db.json
+    # to force next image to load minigraph.
+    if path.exists("/host/old_config/minigraph.xml"):
```

Comment:
Is this supposed to be minigraph.xml or config_db.json?

Reply:
Would there be a case where this does not exist? Then we will still run into the issue?

Reply:
yes. if the system is configured by config_db.json directly, this change cannot address that. I should add this as a known limitation in PR comments.

Reply:
This is my question as well. Is there a scenario where we would *want* to have the old config_db.json?

Reply:
oops, didn't refresh, we're good here. :P